### PR TITLE
COMPLETED!! Iteration5

### PR DESCRIPTION
**Pull request by mcat56**
@sasloan   @zacisaacson @freeheeling

### CODE REVIEW

**Comment by zacisaacson** [comment]
Yeeeeessssssssss!!!!!!!

File: data/dummy_game_teams.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
-2012030221,3,away,LOSS,OT,John Tortorella,2,8,44,8,3,0,44.8,17,7
```
Comment (freeheeling) [inline data/dummy_game_teams.csv:2]: Should we swap these back?

File: lib/season_stats.rb [hunk 2]
```diff
@@ -1,22 +1,142 @@
...
 
-#sort by their post season
+  def biggest_surprise(season)
+    max = generate_difference(season).max_by {|team, difference| difference }[0]
+    @teams[max].teamName
+  end
 
-# find how many games they won
+  def generate_wins_by_coach(season)
+    coach_wins = Hash.new(0)
```
Comment (zacisaacson) [inline lib/season_stats.rb:33]: Looks great!!!!

**Review by zacisaacson — APPROVED** [review]

**Review by freeheeling — APPROVED** [review]
Approved.

**Pull request merged**